draft-irtf-cfrg-hpke.md

```diff
@@ -847,12 +847,11 @@ rejection sampling over field elements:
 
 ~~~
 def DeriveKeyPair(ikm):
-  prk = LabeledExtract(zero(0), I2OSP(kem_id, 2), ikm)
+  dkp_prk = LabeledExtract(zero(0), concat(I2OSP(kem_id, 2), "dkp_prk"), ikm)
```

Comment:
Is there a downside of putting the kem_id in the salt parameter and omitting the concat operation? It'd be cleaner

Reply:
Yeah, I wondered that myself when making the change.

Reply:
[The `salt` parameter is supposed to be random](https://tools.ietf.org/html/rfc5869#section-2.2) and [independent of `ikm`](https://tools.ietf.org/html/rfc5869#section-3.4), so I believe we should not use it to put the constant `kem_id` there.

Reply:
Yeah, it would be "misuse", but I don't think it makes any difference internally. Anyway, fine to leave this as is!